tests/integration/test_schema_registry_auth.py

```diff
@@ -170,34 +188,36 @@ async def test_sr_ids(registry_async_client_auth: Client) -> None:
     jsonschema_sc_id = res.json()["id"]
     assert jsonschema_sc_id >= 0
 
-    res = await registry_async_client_auth.get(f"schemas/ids/{avro_sc_id}", auth=aladdin)
+    res = await registry_async_retry_client_auth.get(f"schemas/ids/{avro_sc_id}", auth=aladdin)
     assert res.status_code == 200
 
-    res = await registry_async_client_auth.get(f"schemas/ids/{jsonschema_sc_id}", auth=aladdin)
+    res = await registry_async_retry_client_auth.get(
+        f"schemas/ids/{jsonschema_sc_id}", auth=aladdin, expected_response_code=404
+    )
     assert res.status_code == 404
     assert {"error_code": 40403, "message": "Schema not found"} == res.json()
 
-    res = await registry_async_client_auth.get(f"schemas/ids/{avro_sc_id}", auth=reader)
+    res = await registry_async_retry_client_auth.get(f"schemas/ids/{avro_sc_id}", auth=reader, expected_response_code=404)
     assert res.status_code == 404
     assert {"error_code": 40403, "message": "Schema not found"} == res.json()
 
-    res = await registry_async_client_auth.get(f"schemas/ids/{jsonschema_sc_id}", auth=reader)
+    res = await registry_async_retry_client_auth.get(f"schemas/ids/{jsonschema_sc_id}", auth=reader)
     assert res.status_code == 200
 
 
-async def test_sr_auth_forwarding(registry_async_auth_pair: List[str]) -> None:
-    auth = requests.auth.HTTPBasicAuth("admin", "admin")
-
+async def test_sr_auth_forwarding(
+    registry_async_auth_pair: List[str], registry_async_retry_client_auth: RetryRestClient
+) -> None:
     # Test primary/replica forwarding with global config setting
     primary_url, replica_url = registry_async_auth_pair
     max_tries, counter = 5, 0
     wait_time = 0.5
     for compat in ["FULL", "BACKWARD", "FORWARD", "NONE"]:
-        resp = requests.put(f"{replica_url}/config", json={"compatibility": compat}, auth=auth)
+        resp = await registry_async_retry_client_auth.put(f"{replica_url}/config", json={"compatibility": compat}, auth=admin)
```

Comment:
This was failing and we had to split the used `auth`, `aiohttp` requires the `BasicAuth` and `requests` requires something different, but we were first passing the requests auth to the aiohttp client and it was failing.